mangaki/mangaki/utils/anidb.py

```diff
@@ -93,7 +93,7 @@ def lang_map(self) -> Dict[str, ExtLanguage]:
 
     @cached_property
     def unknown_language(self) -> ExtLanguage:
-        return ExtLanguage.objects.get(source='anidb', ext_lang='x-unk')
+        return ExtLanguage.objects.filter(source='anidb', ext_lang='x-unk').first()
```

Comment:
Theorically a data migration should populate this.
Hmm. I'll look on my side and get back on this. Keep it as-is for now so you can work on the PR, I'll ping for the appropriate resolution.